src/proj_4D_api.c

```diff
@@ -794,7 +794,12 @@ PJ  *proj_create_crs_to_crs (PJ_CONTEXT *ctx, const char *srid_from, const char
         return NULL;
     }
 
-    P = proj_create(ctx, proj_string);
+    if( proj_string[0] == '\0' ) {
+        /* Null transform ? */
```

Comment:
Maybe we should add a proper null operation? We can call it `noop`.

Reply:
Traced as https://github.com/OSGeo/proj.4/issues/1185 . Low priority